sources/Cdn/Cdn.csproj

```diff
@@ -1,17 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
     <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
+        <LangVersion>10</LangVersion>
```

Comment:
we should change this to either `latest` or `preview` (I'd prefer preview, as it's just the language version, how bad could it break? :when:)

Reply:
This is fine tho, we can bump as newer versions release, and it also keeps versions better controller (imagine if your lang version were a minor or something newer than mine or Favna's, that'd lead to some funny errors).

Reply:
Hm, this is a good point, unless we all kept our .NET version synced, that could be an issue.